sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2267,6 +2267,215 @@ case class FormatNumber(x: Expression, d: Expression)
   override def prettyName: String = "format_number"
 }
 
+/**
+ * A function that converts string to numeric.
+ */
+// scalastyle:off line.size.limit
```

Comment:
The code below doesn't reach the limit. This can be removed.

Reply:
Thanks, I will follow your suggestion.